test/Engine/ProtoTestConsoleRunner/ProtoTestConsoleRunner.csproj

```diff
@@ -14,12 +14,9 @@
   <PropertyGroup>
     <StartupObject>ProtoTestConsoleRunner.Program</StartupObject>
   </PropertyGroup>
-  <PropertyGroup>
-    <SignAssembly>false</SignAssembly>
-  </PropertyGroup>
-  <PropertyGroup>
-    <AssemblyOriginatorKeyFile>ProtoTestConsoleRunner.snk</AssemblyOriginatorKeyFile>
-  </PropertyGroup>
```

Comment:
Is this removed because we don't need to sign test assemblies?

Reply:
We have not used these files or this signing method for many years.